server/lib/matchmaking/models.ts

```diff
@@ -418,6 +418,52 @@ export async function getManyMatchmakingRatings(
   }
 }
 
+/**
+ * Returns a list of players's finalized matchmaking ranks, and optionally filtered by a search
+ * query, for a particular matchmaking type and season.
+ */
+export async function getManyMatchmakingFinalizedRanks(
```

Comment:
This is a kind of weird naming for this, we typically only use this pattern when we're passing in a bunch of IDs to retrieve, and that's not what this method is doing. I'd probably call this `getFinalizedRanksForSeason` or something